AWS API Module get timeout fix (#16956)
demisto/content: Packs/AWS-ACM/ReleaseNotes/1_1_5.md

> ##### AWS-ACM

Reviewer: Follow the release notes template in all files
https://xsoar.pan.dev/docs/documentation/release-notes#md-formatting

> - Fixed an issue where the timeout parameter couldn't be of type *int*.

Reviewer 2: ```suggestion
- Fixed an issue where the timeout parameter would not support an integer.
```
 Apply this to the others as well.
